lib/query.js

```diff
@@ -45,6 +45,22 @@ var detectGroupType = function(group) {
     }) || "";
 };
 
+var splitGroup = function(group) {
+  var parts = group.split(FILTER_DELIMITER);
+  var field = removeQuotation(parts[0]);
+  var query = removeQuotation(parts.slice(1).join(FILTER_DELIMITER));
+  var type = detectGroupType(query);
+  var value = query.slice(type.length);
+
+  return {
+    length: parts.length,
+    field: field,
+    query: query,
+    type: type || "=",
+    value: value
+  }
+}
+
```

Comment:
This is just a quick refactoring of [L57-62 of query.js](https://github.com/GitbookIO/filterable/blob/master/lib/query.js#L57-L62) so that I may reuse it during this implementation.